modules/file-system/parallelstore/scripts/mount-daos.sh

```diff
@@ -41,6 +41,17 @@ sed -i "s/#.*transport_config/transport_config/g" $daos_config
 sed -i "s/#.*allow_insecure:.*false/  allow_insecure: true/g" $daos_config
 sed -i "s/.*access_points.*/access_points: $access_points/g" $daos_config
```

Comment:
Does the goal here is only to support multiple mounts from the server Parallelstore instance, or to support mounts from multiple Parallelstore instances?

I fear, that with multiple Parallelstore instances, the last script that will run will setup `access_points` in the config, and it will be the only instance mounted (but multiple times)

Reply:
The goal is to have 1 parallelstore instance but able to mount to multiple mount points (possible with different mount options).

Reply:
Ok, perfect. I think it is worth mentioning in the README, that module supports mounting only from Parallelstore instance, but it's possibile to mount it multiple times with different options using `pre-existing-network-storage`.

Probably worth mentioning in both modules.

Reply:
We will review documentation as well as potential for multiple parallelstore mount functionality.